### PR TITLE
Trg changes part 2

### PR DESCRIPTION
**Pull request by markkramerus**
The TRG (particularly Rich Moldwin) has recommended transitioning the identifying codes to SNOMED. At the time I didn't understand how to do this in a backward-compatible way, but the max value set pattern gives us a feasible path. This PR includes transitions to SNOMED on the TNM staging codes. We probably need a JIRA issue to make this change legit --> https://jira.hl7.org/browse/FHIR-41655

### CODE REVIEW

File: input/fsh/VS_Staging.fsh [hunk 1]
```diff
@@ -5,34 +5,8 @@ ValueSet: CancerStageTypeVS
...
-* ^copyright = "This value set includes content from SNOMED CT, which is copyright © 2002+ International Health Terminology Standards Development Organisation (IHTSDO), and distributed by agreement between IHTSDO and HL7. Implementer use of SNOMED CT is not covered by this agreement. This material contains content from LOINC (http://loinc.org). LOINC is copyright © 1995-2020, Regenstrief Institute, Inc. and the Logical Observation Identifiers Names and Codes (LOINC) Committee and is available at no cost under the license at http://loinc.org/license. LOINC® is a registered United States trademark of Regenstrief Institute, Inc"
-* ^experimental = false
+* insert SNOMEDCopyrightForVS
 * ^extension[FMM].valueInteger = 4
-// LOINC Codes
-* LNC#21907-1 "Distant metastases.clinical [Class] Cancer"
```
Comment (saulakravitz) [inline input/fsh/VS_Staging.fsh:12]: Seems like in some places you are banishing LOINC codes, and in other place you are including them in maxVS...   I don't understand the banishment, but why not take a consistent approach to deprecation?

File: input/fsh/DEF_RuleSets.fsh [hunk 2]
```diff
@@ -51,6 +51,10 @@ RuleSet: LOINCCopyrightForVS
 * ^copyright = "This material contains content from LOINC (http://loinc.org). LOINC is copyright © 1995-2020, Regenstrief Institute, Inc. and the Logical Observation Identifiers Names and Codes (LOINC) Committee and is available at no cost under the license at http://loinc.org/license. LOINC® is a registered United States trademark of Regenstrief Institute, Inc"
 * ^experimental = false
 
+RuleSet: SNOINCCopyrightForVS
```
Comment (saulakravitz) [inline input/fsh/DEF_RuleSets.fsh:54]: Should there be different rulesets for the copyright notice for LOINC-based VS,  SNOMEDCT-based VS, and LOINC+SNOMEDCT-based valuesets?

**Review by saulakravitz — DISMISSED** [review]

File: input/fsh/VS_Staging.fsh (hunk 1 above)
Comment (markkramerus) [reply input/fsh/VS_Staging.fsh:12]: Including them in the maxVS is exactly how to "banish" them.

File: input/fsh/DEF_RuleSets.fsh (hunk 2 above)
Comment (markkramerus) [reply input/fsh/DEF_RuleSets.fsh:54]: Well, yes, this is what is happening here... SNOINC = SNOMED + LOINC

**Pull request merged**